integration/gradle-plugin-integration-tests/client-generator/custom-scalars-kotlinx/src/test/kotlin/com/expediagroup/scalars/CustomScalarKotlinxTests.kt

```diff
@@ -31,11 +31,11 @@ class CustomScalarKotlinxTests {
 
     @Test
     fun `verify custom scalars are correctly serialized and deserialized`() {
-        val engine = embeddedServer(CIO, port = 0, module = Application::graphQLModule)
+        val embeddedServer = embeddedServer(CIO, port = 0, module = Application::graphQLModule)
```

Comment:
`embeddedServer()` now returns an `EmbeddedServer` which contains the `ApplicationEngine`.
https://ktor.io/docs/migrating-3.html#EmbeddedServer